Ginger/Ginger/AutomatePageLib/AddActionMenu/ActionsLibrary/ActionsLibraryNavPage.xaml.cs

```diff
@@ -172,10 +174,24 @@ private void LoadGridData()
                 }
             }
 
-            //xPlatformActionsListView.DataSourceList = platformActions;
-            PlatformActionsGrid.DataSourceList = platformActions;
-            GeneralActionsGrid.DataSourceList = generalActions;
-            LegacyActionsGrid.DataSourceList = LegacyActions;
+            SetActionsListView(xPlatformActionsListView, platformActions);
+            SetActionsListView(xPlatformGenericActionsListView, generalActions);
+            SetActionsListView(xPlatformLegacyActionListView, LegacyActions);
+        }
+
+        private void SetActionsListView(UcListView ucListView, ObservableList<Act> dataSource)
+        {
+            ucListView.ListTitleVisibility = Visibility.Hidden;
+            mActionsListHelper = new ActionsLibraryListViewHelper(mContext, mPageViewMode);
+
+            ucListView.SetDefaultListDataTemplate(mActionsListHelper);
+            ucListView.ListSelectionMode = SelectionMode.Extended;
+            ucListView.DataSourceList = dataSource;
+            mActionsListHelper.ListView = ucListView;
+
+            ucListView.MouseDoubleClick -= ActionsListView_MouseDoubleClick;  // In order to prevent adding multiple time of the event
```

Comment:
Regular Actions list (under Activity in Automate page) is also supporting double click (opening edit page)- also there the even was added same as you done here?